spring-jdbc/src/main/java/org/springframework/jdbc/support/KeyHolder.java

```diff
@@ -58,6 +58,22 @@
 	@Nullable
 	Number getKey() throws InvalidDataAccessApiUsageException;
 
+	/**
+	 * Retrieve the first item from the first map, assuming that there is just
+	 * one item and just one map, and that the item is an instance of provided class.
+	 * This is the typical case: a single, numeric generated key.
```

Comment:
The above line is still confusing. For such typical case, should we use the previous method returning Number instead?

Reply:
fixed as well.